lib/glimesh/graphql/schema/account_types.ex

```diff
@@ -41,10 +41,10 @@ defmodule Glimesh.Schema.AccountTypes do
       config(fn args, _ ->
         case Map.get(args, :streamer_id) do
           nil ->
-            {:ok, topic: [Glimesh.AccountFollows.get_subscribe_topic(:follows)]}
+            {:ok, topic: [get_subscribe_topic(:follows)]}
```

Comment:
Sorry, should've been more clear. Still getting used to doing proper reviews. When adding an alias it basically removes the necessity for the Glimesh. part, you'll still want everything after, so for this instance, it's 
```suggestion
            {:ok, topic: [AccountFollows.get_subscribe_topic(:follows)]}
```